src/cadenza.js

```diff
@@ -432,6 +432,19 @@ export class CadenzaClient {
     return this.#show(resolvePath(backgroundMapView), params, signal);
   }
 
+  /**
+   * Set the visibility of a layer in an embedded map view. The map view must be shown prior to calling this.
+   *
+   * @param {WorkbookLayerPath} layer - The layer to show or hide, identified by its path
+   * @param {boolean} visible - The visibility state of the layer
+   */
+  setLayerVisibility(layer, visible = true) {
+    this.#log('CadenzaClient#setLayerVisibility', ...arguments);
+    if (layer) {
+      this.#postEvent('setLayerVisibility', { layer, visible });
```

Comment:
As you suggested we should use `#postRequest` here instead.

Reply:
Background: #postEvent is for sync, where we do not expect errors, #postRequest for async, where we need to know whether it worked. Since here we execute a workbook command in Cadenza, #postRequest is the right tool.